src/helpers/RecordsProcessor.js

```diff
@@ -28,7 +28,7 @@ const processRecords = function (schema, records, outputFormat) {
         }
         resultString += (value + '|')
       })
-      resultString = resultString.slice(0, -1)
+      resultString = resultString.slice(0, -1) + '\n'
```

Comment:
You go to great effort to avoid using Array `join`!

Reply:
Yeah... I need to join by '|' but I also have to check each value for '|' and I just figured if I'm already iterating through each value I may as well join them myself!